apps/system/test/marionette/net_error_test.js

```diff
@@ -2,11 +2,7 @@ var assert = require('assert');
 var APP_FAKE = 'http://fake.fake.fake';
 
 marionette('net_error.html:', function() {
-  var client = marionette.client({
-    settings: {
-      'ftu.manifestURL': null
```

Comment:
Not in this case, the test appears to append the content itself, so lockscreen state doens't matter.